lib/Firefox/FirefoxProfile.php

```diff
@@ -27,6 +28,16 @@ class FirefoxProfile
      */
     private $rdf_file;
 
+    /**
+     * Return a version of the class which can JSON serialized.
+     *
+     * @return  array
+     */
+    public function jsonSerialize()
+    {
+        return $this->encode();
```

Comment:
But this returns string - while it must return something serializable using `json_encode()`. It also means we don't have a test for this (otherwise it would fail). And we should increase phpstan level, the current level 2 one does not check for this 😬.

But do we actually need JsonSerializable on FirefoxProfile? I believe the issue in #849 is just with ChromeOptions and FirefoxProfile is not related.

Reply:
Yes - you're right. I misunderstood the purpose of FirefoxProfile. Killing that change.